client/material/tabs.tsx

```diff
@@ -174,5 +177,52 @@ export function Tabs<T>({ children, activeTab, onChange, className }: TabsProps<
     return tabElems
   }, [activeTab, children, onChange])
 
+  useKeyListener({
+    onKeyDown: (event: KeyboardEvent) => {
+      if ((event.keyCode === PAGEUP || event.keyCode === PAGEDOWN) && event.ctrlKey === true) {
+        const enabledChildren: React.ReactElement[] = []
+        React.Children.forEach(children, (child, i) => {
+          if (child && !(child.props as TabItemProps<T>).disabled) {
+            enabledChildren.push(child)
+          }
+        })
+
+        if (enabledChildren.length < 2) {
+          return false
```

Comment:
Feel like this should probably return true in this case: The hotkey was right but we're not in a state where it would do anything, so we're exiting early